packages/fetch/src/headers.js

```diff
@@ -138,7 +138,7 @@ export default class Headers extends URLSearchParams {
 							validateHeaderName(name);
 							// @ts-ignore
 							return URLSearchParams.prototype[p].call(
-								receiver,
+								target,
 								String(name).toLowerCase()
```

Comment:
Looks like this introduces a regression in tests see https://github.com/web-std/io/actions/runs/5986735537/job/16286188008?pr=86

Reply:
@mcansh Any idea what this could have caused?

Reply:
narrowed it down to internal version mismatches between packages

Reply:
closer! https://github.com/web-std/io/actions/runs/6043044508/job/16399344526?pr=99#step:5:12

Reply:
That would have been fixed if #74 was already merged then

@Gozala another reason to _at least_ include `manypkg` (but I would do the switch to `changeset` as well) 🙈